packages/shorebird_cli/lib/src/shorebird_flutter.dart

```diff
@@ -6,6 +6,7 @@ import 'package:path/path.dart' as p;
 import 'package:pub_semver/pub_semver.dart';
 import 'package:scoped_deps/scoped_deps.dart';
 import 'package:shorebird_cli/src/executables/executables.dart';
+import 'package:shorebird_cli/src/executables/flutter_version.dart';
```

Comment:
This import shouldn't be needed if we export `flutter_version` from `executables/executables.dart`

Also the convention would be to create a class called `Flutter` under executables which exposes whatever we need from the underlying `flutter` tool. This is what the `ShorebirdFlutter` class was meant to do.